packages/canary-web-components/src/components/ic-tree-item/ic-tree-item.css

```
@@ -37,6 +37,16 @@
   outline: none;
 }
 
+:host(.ic-tree-item-focus-inset) .tree-item-content:focus,
+:host(.ic-tree-item-focus-inset) ::slotted([slot="router-item"]:focus) {
+  box-shadow: var(--ic-border-focus-inset);
```

I think it would be good to apply a border radius so that the corners are rounded like other focus indicators
```suggestion
  box-shadow: var(--ic-border-focus-inset);
  border-radius: var(--ic-border-radius-inset);
```